sinonjs/sinon
# Fix spy documentation and fix contributing typo

#### Purpose (TL;DR) - mandatory
* Fix `matchers` link in `spies.md` and `spy-call.md` across all versions
* Fix typo in `docs/CONTRIBUTING`

#### Background (Problem in detail)  - optional
When I click `see matchers`, I would expect the link to take me to the `matchers` page, but this instead sets an anchor that doesn't exist on the page. Correct me if I'm missing something here.

In addition, the links don't work in Github's markdown viewer, but I assume something with how the doc paths are reconciled makes this work in the actual doc viewer. To try and ensure the correct path, I took a look at the `[calls]: ../spy-call` link which appears to work in the actual documentation. Let me know if this is not the case and I'll be happy to update. :smile: 

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. Validate docs for `spies.md` and `spy-call.md` from the `v1.17.6-v6.1.3`

#### Checklist for author

  - NOTE: One warning, but it looks unrelated: 
```
sinon/test/sandbox-test.js
  217:34  warning  Do not create function within loops (IE11 compatibility)  ie11/no-loop-func
```

![](https://media.giphy.com/media/toXKzaJP3WIgM/giphy.gif)
